service/service.go

```diff
@@ -40,6 +40,14 @@ func CombinedClientInfoHandler(r *http.Request) (clientID, clientSecret string,
 	return
 }
 
+func AccessTokenExpHandler(w http.ResponseWriter, r *http.Request) (exp time.Duration, err error) {
+	expiry, err := strconv.Atoi(r.FormValue("expiry"));
+	if err != nil {
+		return time.Duration(0), err
+	}
+	return time.Duration(expiry) * time.Second, nil
```

Comment:
We originally intended to pass one field `expires_at` which is a unix timestamp in seconds.

Here it is called `expiry` and only receives the number of seconds until expiry. Could it be updated here? or should we change where we are doing the request? CC @bumi

Reply:
Changing it to be `expires_at ` then

Reply:
whatever is easier. 

in ruby that would be `1.year.to_i`

Reply:
Done!